### DISCUSSION TITLE
SearchableText not working on id column

### DISCUSSION THREAD

**Discussion by huynguyen-twentyci** [laravel/nova-issues#5912] [Q&A]
- Laravel Version: 10.24.0
- Nova Version: 4.27.13
- PHP Version: 8.2.9
- Database Driver & Version: MSSQL 17
- Operating System and Version: macOS 13.5.2
- Browser type and version: Chrome 117.0.5938.92
- Reproduction Repository: https://github.com/###/###

### Description:
I'm using nova 4. I just want to make my id column on a resource to a full-text search ("like" in sql), just like searching "1000" is fine with result "10001, 10002". I found a method SearchableText, but obviously it's not working on id column, it's working only on the other fields than id.

### Detailed steps to reproduce the issue on a fresh Nova installation:
- Install laravel and nova
- Create a new resource with columns "id", "name" and insert some data
- Try to make id column searchable on nova with "like" condition, or full-text search...etc.
For example id 10001, 10002...then search with 1000 is ok

**Comment by crynobone** [comment]
You should be able to customize, or build custom queries: https://twitter.com/ganyicz/status/1661424908000165891
